src/common/hooks/useAxios.ts

```diff
@@ -61,13 +56,33 @@ export default () => {
             });
     }
 
-    async function patchRequest<RETURN, BODY>(
+    async function patchRequest<Return, Body>(
         url: string,
-        body?: BODY
-    ): Promise<RETURN | ErrorResponse> {
+        body?: Body
+    ): Promise<Return | ErrorResponse> {
         const token = await getAccessTokenSilently();
         return instance
-            .patch<RETURN>(url, body, {
+            .patch<Return>(url, body, {
+                headers: {
+                    "Content-Type": "application/json",
+                    Authorization: `Bearer ${token}`,
+                },
+            })
+            .then((res) => res.data)
+            .catch((err) => {
```

Comment:
It would be nice is this was written as
`.catch(handleError)` instead, but I think you might run into type issues with that, but if you find a way, that would be nice. But at minimum you could do 
```
.catch((err) => handleError(err as AxiosError)
```

Reply:
apply throughout